tensorflow/compiler/aot/compile.cc

```diff
@@ -176,6 +176,10 @@ static void InitializeTargets() {
   LLVMInitializePowerPCTargetInfo();
   LLVMInitializePowerPCTargetMC();
   LLVMInitializePowerPCAsmPrinter();
+  LLVMInitializeSystemZTarget();
+  LLVMInitializeSystemZTargetInfo();
+  LLVMInitializeSystemZTargetMC();
+  LLVMInitializeSystemZAsmPrinter();
```

Comment:
This needs to be `#ifdef` guarded, like `TF_LLVM_AARCH64_AVAILABLE`.

Reply:
This should be solved now.